container/image_test.py

```diff
@@ -760,6 +760,185 @@ def test_war_image(self):
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/module.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/net.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/os.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/package.json', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/path.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/perf_hooks.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/process.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/punycode.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/querystring.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/readline.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/repl.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/stream.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/string_decoder.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/timers.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/tls.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/trace_events.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2/globals.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2/index.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2/util.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/tty.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/url.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/util.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/v8.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/vm.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/worker_threads.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/zlib.d.ts', 
+        '/app', 
+        '/app/testdata', 
+        '/app/testdata/nodejs_image_binary', 
+        '/app/testdata/nodejs_image_binary.runfiles', 
+        '/app/testdata/nodejs_image_binary.runfiles/io_bazel_rules_docker', 
+        '/app/testdata/nodejs_image_binary.runfiles/io_bazel_rules_docker/external'
```

Comment:
If you add `--build noexternal_legacy_runfiles` to `.bazelrc` the `/app/testdata/nodejs_image_binary.runfiles/io_bazel_rules_docker/external` should away which would be preferred for this test and should be the default in Bazel at some point. I was hoping they would flip that for 1.0 but they didn't yet.

Reply:
hmmm I cant see any trace of that flag throughout the repository, ill add a issue then put a TODO to remove these entries since im sure it'll impact other containers and their tests

Reply:
https://github.com/bazelbuild/rules_docker/issues/1350

Reply:
👍